### PR TITLE
By popular demand, walrus fact contractions are now possible.

### PR DESCRIPTION
**Pull request by EgMan**
I have not ran or tested this ¯\\_(ツ)_/¯

### CODE REVIEW

File: walrus_facts.py [hunk 1]
```diff
@@ -90,11 +92,14 @@ def is_handlable_request(requests):
 ##                    if 'user' in request and request['user'] != BOT_ID:
 ##                        pprint(requests)
 ##                        return True
+                contract = False
+                if 'contract' in request:
```
Comment (richardsonj) [inline walrus_facts.py]: What if I just want to ask about contracts?

@walrus_facts how do you feel about negotiating contracts?

Comment (EgMan) [reply walrus_facts.py]: Having it just contract has nothing to do with walruses lol

Comment (richardsonj) [reply walrus_facts.py]: Nevermind. We're not parsing the message, we're just deciding whether or not to contract the fact

Comment (EgMan) [reply walrus_facts.py]: wait, it might need to be something like:
```
if 'contract' in request['text'].lower():
```

File: walrus_facts.py [hunk 2]
```diff
@@ -90,11 +92,14 @@ def is_handlable_request(requests):
...
+                    contract = True
                 if 'user' in request and request['user'] == WALRUS_USER_ID:
-                    return True
+                    return True, contract
                 if 'text' in request and AT_BOT in request['text'] \
                    and 'fact' in request['text'].lower():
-                    return True
+                    return True, contract
     return False
```
Comment (richardsonj) [inline walrus_facts.py]: add the second return value

**Pull request merged**